### PR TITLE
Code golf compat/suspense-list and diff/props

### PR DESCRIPTION
**Pull request by theZieger**

### CODE REVIEW

File: src/diff/props.js [hunk 1]
```diff
@@ -55,16 +55,15 @@ function setStyle(style, key, value) {
  * @param {boolean} isSvg Whether or not this DOM node is an SVG node or not
  */
 function setProperty(dom, name, value, oldValue, isSvg) {
-	if (isSvg) {
-		if (name === 'className') {
-			name = 'class';
-		}
-	} else if (name === 'class') {
+	if (name === 'key' || name === 'children') return;
```
Comment (JoviDeCroock) [inline src/diff/props.js]: return is a non-minified keyword so this most of the time will introduce extra code compared to making one big if else branch

File: compat/test/browser/render.test.js [hunk 2]
```diff
@@ -190,7 +190,6 @@ describe('compat render', () => {
 		update();
 		rerender();
 
-		console.log(scratch.textContent);
```
Comment (JoviDeCroock) [inline compat/test/browser/render.test.js]: Good spot!

File: compat/src/suspense-list.js [hunk 3]
```diff
@@ -30,10 +30,8 @@ const resolve = (list, child, node) => {
 	// callbacks won't get queued in the node anyway.
 	// If revealOrder is 'together' then also do an early exit
 	// if all suspended descendants have not yet been resolved.
-	if (
-		!list.props.revealOrder ||
-		(list.props.revealOrder[0] === 't' && list._map.size)
-	) {
+	const revealOrder = list.props.revealOrder;
```
Comment (JoviDeCroock) [inline compat/src/suspense-list.js]: Most of the time assigning won't save bytes unless used 3+ times (var adds 3bytes)

**Comment by theZieger** [comment]
@JoviDeCroock thanks for the fast response ... been hoping for the bot comment to get the real result on the golfing ... could only compare the output of `npm run build` locally so I was hoping to get a better insight from the bot ... should maybe mark this as WIP.

**Comment by JoviDeCroock** [comment]
You can check it here: https://github.com/preactjs/preact/pull/2303/checks?check_run_id=424170676

**Comment by theZieger** [comment]
What a bummer. is there a reliable way to look this up locally?

**Comment by JoviDeCroock** [comment]
Most of the time I just do `yarn build:<target>` and compare with changes

**Comment by theZieger** [comment]
Bildschirmfoto von 2020-02-03 23-23-44
left: my pr
right: master

At least the changes to compat look still slightly better.
Don't know what I compared regarding preact locally :man_shrugging:
Should maybe revert only those...

**Comment by theZieger** [comment]
Ok it's compat.module is in fact increasing. I'm just to stupid to compare numbers ...

**Comment by JoviDeCroock** [comment]
Your module build is 4b larger on the left

**Comment by theZieger** [comment]
Yeah. Still don't know what I compared locally ... should just pull to terminals up instead of switching back and forth between branches ...
will close this and open one for the console.log in a sec.

**Pull request closed without merging**